Signum.Engine.Extensions/Mailing/EmailLogic.cs

```diff
@@ -41,7 +41,7 @@ internal static void AssertStarted(SchemaBuilder sb)
         public static void Start(
             SchemaBuilder sb,  
             Func<EmailConfigurationEmbedded> getConfiguration, 
-            Func<EmailTemplateEntity?, Lite<Entity>?, EmailSenderConfigurationEntity> getEmailSenderConfiguration,  
+            Func<EmailTemplateEntity?, Entity?, EmailSenderConfigurationEntity> getEmailSenderConfiguration,  

```

Comment:
We changed this from ModifiableEntity to Lite<Entity> two years ago (https://github.com/signumsoftware/extensions/commit/7f01a66e96505279604614624a0cfd0af0465351). Now you want to change to Entity but passing the EmailMessageEntity (instead of the target) *sometimes*. Looks fishy. Can you give me more details?

Reply:
Our determination of the EmailSenderConfigurationEntity sometimes depends on the tarject or template and for performance it is faster if the message is new or saved in the database. therefore we need the message which is present at the time of invoking the method then there is no loss of performance or information in the change